BepInEx.Core/Bootstrap/LoadingSystem.cs

```diff
@@ -154,7 +182,7 @@ protected static bool PluginTargetsWrongBepin(PluginInfo pluginInfo)
     ///     Collection of error chainloader messages that occured during plugin loading.
     ///     Contains information about what certain plugins were not loaded.
     /// </summary>
-    public List<string> DependencyErrors { get; } = new();
+    private List<string> DependencyErrors { get; } = new();
```

Comment:
This needs to be public for https://github.com/BepInEx/BepInEx.Utility/blob/master/BepInEx.MessageCenter/MessageCenter.cs#L33-L34

Reply:
Decided to make public for now, but am open to suggestions to improve on this